lib/mods/replace-window-functions.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const nodeTypes = require('ts-morph').TypeGuards;
+const {updateImports} = require('../imports.js');
+const {
+  formatReplacement,
+  getOriginal,
+} = require('../originals-helper.js');
+
+/**
+ * @param {*} node - the node to examine
+ *
+ * @return {boolean} whether the node represents a window function
+ */
+function isWindowProperty(node) {
+  const symbol = node.getExpression().getSymbol();
+  const firstDeclaration = (symbol) ? symbol.getDeclarations()[0] : undefined;
+
+  // TODO: may require a deeper check
+  // e.g. setTimeout web returns number, Node returns Timer
```

Comment:
This doesn't really clarify it for me. We're never operating in Node, only the web. What is a case where firstDeclaration is undefined vs. not?

Reply:
I think I just got confused here, it seemed too easy (because it was, I can't just use `window` the way I currently am)